lib/ansible/modules/user.py

```diff
@@ -52,7 +52,7 @@
         type: str
     groups:
         description:
-            - List of groups user will be added to.
+            - List of SECONDARY groups user will be added to.
```

Comment:
```suggestion
            - A list of supplementary groups which the user is also a member of.
```
from the useradd manpage for the `--groups` option